presto-docs/src/main/sphinx/release/teradata-additions.rst

```diff
@@ -4,18 +4,12 @@ Teradata Additions
 
 The following are additions that Teradata has added to Facebook's 0.127 release of presto
 
-General Fixes
--------------
-
-* Remove buggy optimization to prune redundant projections because it produced wrong results.
-
-Datatypes
```

Comment:
Got rid of this because it's probably no longer as experimental.  I'll send out a note to eng that people should add release notes here for the features they've added.